File: DotNetGrc/Grc/Ast/Node/Type/TypeDataBase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Grc.Ast.Visitor;

namespace Grc.Ast.Node.Type
{
	public abstract class TypeDataBase : TypeReturnBase
	{
		public TypeDataBase(string text) : base(text)
		{
		}

		public override void Accept(IVisitor v)
		{
			v.Visit(this);
		}
	}
}

File: Parser/src/main/java/k31/grc/ast/node/type/TypeDataBase.java
package k31.grc.ast.node.type;

import k31.grc.ast.visitor.Visitor;

public abstract class TypeDataBase extends TypeReturnBase {

	public TypeDataBase(String text) {
		super(text);
	}

	@Override
	public void accept(Visitor v) {

		v.visit(this);
	}
}
